ginga/rv/plugins/MultiDim.py

```diff
@@ -288,6 +304,7 @@ def play_axis_change():
                         self.play_max = dims[self.play_axis]
 
                     self.play_idx = self.play_indices[n - 2]
+                    self.play_last[imname] = self.play_idx
```

Comment:
If you look at the `set_naxis()` method it is doing a
```python
image.set_naxispath(self.naxispath)
```
which means that the representation inside `MultiDim` of `naxispath` is compatible with that of the image.  So in `redo()` I think we can do something like
```python
naxispath = image.naxispath.copy()
```
and then set the controls from that.